.github/workflows/ci.yml

```diff
@@ -144,11 +144,10 @@ jobs:
       - name: Push to GAR
         uses: grafana/shared-workflows/actions/push-to-gar-docker@main
         with:
-          # Only push to GAR on main branch pushes
-          push: ${{ github.event_name == 'push' && github.ref_name == 'main' && 'true' || 'false' }}
+          push: true
```

Comment:
I guess these should be reverted either directly before or directly after merging this 🙂

Reply:
yeah, sorry, we should have opened this as a draft. we are testing things out and try to figure out how to best get decent queue metrics 🙏